instrumentation/net/http/otelhttp/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	go.opentelemetry.io/otel v1.20.0
 	go.opentelemetry.io/otel/metric v1.20.0
+	go.opentelemetry.io/otel/sdk/metric v1.20.0
```

Comment:
The instrumentation library MUST not depend on any SDK implementation. I think you need this only for testing. Then please move the test code under `test` subdirectory to solve the issue.

Reply:
I see. Will do.